lib/console/channels/channel_resolver.ex

```diff
@@ -78,6 +79,8 @@ defmodule Console.Channels.ChannelResolver do
           _ ->
             channel
         end
+
+        Map.put(channel, :alerts, Alerts.get_alerts_by_node(channel.id, "integration"))
```

Comment:
you have to chain the resulting channel from line 72 into this function, otherwise, endpoint doesnt get put in